Client.xcodeproj/project.pbxproj

```diff
@@ -1742,7 +1742,7 @@
 				F84B22541A0920C600AAB793 /* ShareTo.appex in Embed App Extensions */,
 			);
 			name = "Embed App Extensions";
-			runOnlyForDeploymentPostprocessing = 0;
+			runOnlyForDeploymentPostprocessing = 1;
```

Comment:
Nit: Same as above... I did not see a change in this line, if possible I would leave it as it was..